test/e2e/node/pod_resize.go

```diff
@@ -161,39 +165,75 @@ func makeTestPod(ns, name, timeStamp string, tcInfo []TestContainerInfo) *v1.Pod
 	return pod
 }
 
+func makeTestVM(tcInfo TestContainerInfo) v1.VirtualMachine {
+	res, alloc, resizePol := getTestResourceInfo(tcInfo)
+	tvm := v1.VirtualMachine{
+		Name:               tcInfo.Name,
+		KeyPairName:        "foobar",
+		Image:              "download.cirros-cloud.net/0.3.5/cirros-0.3.5-x86_64-disk.img",
```

Comment:
please add a todo. I have another PR to put this image into the e2e test framework in the imageutil class. once the PR is in, we should fix it from this hardcoded image string